demo-apps/cf-plugtest-server/src/main/java/org/eclipse/californium/plugtests/PlugtestServer.java

```diff
@@ -361,6 +376,8 @@ public static Configuration init(BaseConfig config) {
 		// create server
 		try {
 			List<Protocol> protocols = config.getProtocols();
+			protocols.remove(Protocol.DTLS);
```

Comment:
I've tested it with DTLS. Seems to work.
I'm not sure, why it should be removed.

Reply:
Yeah seems to be just a mistake/leftovers from testing. Reverted in commit 082725ddf6e06e789b9535e79a3734af66f2116f

What about TLS? Perhaps I should delete also the following?
https://github.com/eclipse/californium/blob/082725ddf6e06e789b9535e79a3734af66f2116f/demo-apps/cf-plugtest-server/src/main/java/org/eclipse/californium/plugtests/PlugtestServer.java#L379

Reply:
Yes, if the stack-factory gets fixed.

Reply:
Sorry, I may have got it wrong.

No, don't remove neither TCP nor TLS, that is not required!
It is required to add TCP/TLS support to the [OSCoreCoapStackFactory](https://github.com/eclipse/californium/blob/master/cf-oscore/src/main/java/org/eclipse/californium/oscore/OSCoreCoapStackFactory.java#L41), either by using the available [CoapTcpStack](https://github.com/eclipse/californium/blob/master/californium-core/src/main/java/org/eclipse/californium/core/network/CoapEndpoint.java#L1742-L1744) or try to implement a  `OSCoreTcpStack` using the TCP variants of the layers.

And for the client, once the usage of oscore is out in a cli-option, also check the protocol flavor. If TCP is not supported for OSCORE, then just print a warning instead of executing the tests.

But the very most important thing is:
Adapt the  OSCoreCoapStackFactory to support the TCP flavor, even without OSCORE support.

Reply:
Removal of TLS has been deleted now in commit 04e3600364132a1e37a76fa60d5773628254ae1e